### PR TITLE
Inline update user

### PR DESCRIPTION
**Pull request by pketh**
https://inline-update-user.glitch.me/

builds on top of https://github.com/FogCreek/Glitch-Community/pull/84 (this'll be much easier to review after that's merged)
shows an `inline-notification` when a user is invited to join a team. notifications are position:absolute so shouldn't affect page layout

### CODE REVIEW

**Comment by pketh** [comment]
@GregWeil if you approve of this , maybe it makes sense to merge this into your branch, and close this pr

File: src/presenters/pop-overs/add-team-user-pop.jsx [hunk 1]
```diff
@@ -64,6 +64,9 @@ class AddTeamUserPop extends React.Component {
   togglePopoverAnd(func, ...params) {
     this.props.togglePopover();
     func(...params);
+    if (func.name === 'inviteUser' || func.name === 'inviteEmail') {
+      this.props.updateInviteSent()
+    }
```
Comment (GregWeil) [inline src/presenters/pop-overs/add-team-user-pop.jsx]: I don't think we should be identifying the function to call based on its name, it seems pretty fragile. This also means we show the 'sent invitation' inline before we've actually hit the server (and the popup won't correct itself if something goes wrong)

Comment (pketh) [reply src/presenters/pop-overs/add-team-user-pop.jsx]: i can try somethign else. 

re timing, I think it's actually better to show the notification before the 200 comes back. If an error comes back we can display the error, but in 99% of times it's gonna work so assuming success lets us make the ui feel faster

**Comment by pketh** [comment]
@GregWeil split the function into three less clever methods, that each call the inline-notifier and the team-editor

**Comment by pketh** [comment]
now invitees are displayed in the inline notification, replaced the global notification

**Comment by pketh** [comment]
ready for re-review

**Comment by GregWeil** [comment]
Looks good! I guess I'll merge this into superb-guide as an expansion of that review

**Comment by GregWeil** [comment]
Actually, can you clean up the linting errors first?

**Comment by pketh** [comment]
ya sure, one sec

**Comment by pketh** [comment]
ok fixed

**Pull request closed without merging**